### DISCUSSION TITLE
Click item name to do Wiki lookup

### DISCUSSION THREAD

**Discussion by kelltom** [Zoinkwiz/quest-helper#879] [Ideas]
It would be nice if we could click the name of an item in the Quest Helper panel to quickly lookup the item on the Wiki,

For instance, let's say I forget how to obtain climbing boots. I could just click it there and it would bring up the matching Wiki page.

![RuneLite_cPAFB5yDPg](https://user-images.githubusercontent.com/44652363/180101812-5906ff13-38b4-4a2c-8671-f5d6252bf2e9.png)

![chrome_R2lWJ5JICW](https://user-images.githubusercontent.com/44652363/180101938-76dae283-c62b-4945-a5a4-e4a2c09e64b5.png)

**Comment by vmgi** [comment]
I like this idea, I've started working on it.

**Reply by vmgi** [reply]
![itemlinksample](https://user-images.githubusercontent.com/96400383/180582993-f26269d9-94ac-408f-a7cf-a00b9edff3e7.gif)

**Reply by kelltom** [reply]
You legend

**Reply by vmgi** [reply]
#889
